### PR TITLE
Lender reviews

### PR DESCRIPTION
**Pull request by oisinod**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -8,5 +8,12 @@
...
   <p><%= @user.user_name %></p>
   <p><%= @user.location %></p>
+  <% @outgoing_bookings.each do |booking| %>
+   <p> <%= "⭐" * booking.review.rating %>
+       <%= booking.review.content %>
+   </p>
+  <% end %>
   <%= render "shared/grid" %>
+
+
```
Comment (AnjaGberg) [inline app/views/users/show.html.erb:15]: remove line 17 and 18

File: app/views/shared/_review.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    Button with data-target
+  </button>
+<div class="collapse" id="collapseExample">
+  <div class="card card-body">
+    <%= simple_form_for(@review) do |f| %>
+      <%= f.input :rating, collection: 1..5 %>
+      <%= f.input :content %>
+      <%= f.submit %>
+    <% end %>
+
```
Comment (AnjaGberg) [inline app/views/shared/_review.html.erb:11]: remove line

File: app/javascript/controllers/review_form_controller.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Controller } from "@hotwired/stimulus";
+import $ from "jquery";
+import "jquery-bar-rating"
+
+   // import 'select2/dist/css/select2.min.css'; <- Uncomment this line if you are using Rails
+
```
Comment (AnjaGberg) [inline app/javascript/controllers/review_form_controller.js:6]: remove

File: app/javascript/controllers/review_form_controller.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+   export default class extends Controller {
+     connect(){
+       $(function() {
+      $('#review_rating').barrating({
+        theme: 'css-stars',
+        showSelectedRating: false
+    });
+    });
+     }
+
```
Comment (AnjaGberg) [inline app/javascript/controllers/review_form_controller.js:16]: remove

**Review by AnjaGberg — COMMENTED** [review]
just some empty lines

File: app/views/shared/_review.html.erb (hunk 2 above)
Comment (oisinod) [reply app/views/shared/_review.html.erb:11]: done

File: app/views/users/show.html.erb (hunk 1 above)
Comment (oisinod) [reply app/views/users/show.html.erb:11]: done

File: app/javascript/controllers/review_form_controller.js (hunk 4 above)
Comment (oisinod) [reply app/javascript/controllers/review_form_controller.js:16]: done

**Pull request merged**